lib/passport-configurator.js

```diff
@@ -405,6 +406,26 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         }
       );
       break;
+    case 'fb-token':
```

Comment:
You have a field called module, couldn't we modify the generic method to use it, instead of hardcoding it here?